lib/src/store.dart

```diff
@@ -70,7 +70,14 @@ class Store extends Disposable {
         .asBroadcastStream() as Stream<Store>;
   }
 
+  @override
+  bool get isBroadcast => _stream.isBroadcast;
```

Comment:
could add two tests that ensure that this is `true` for both the `new Store()` and `new Store.withTransformer()` constructors